src/app/shared/components/accordion/accordion.component.scss

```diff
@@ -0,0 +1,66 @@
+@import "src/assets/themes/variables";
+
+$edit-col-width: 108px;
+$margin-overlap: 6px;
+$border-width: 2px;
+$border-radius: 4px;
+
+.accordion-header-row {
+  border: $border-width solid $primary-blue;
+  border-radius: $border-radius;
+
+  @media screen and (min-width: 768px) {
```

Comment:
This is a good use case for bootstrap's grid system, as is used by other components in this product. Breakpoints are unified so no hardcoding of pixels is required (typ).

Reply:
The appearance and behaviour of the accordion button is quite custom. I did the best I could to use built-in classes, but there is still a bit of pixel pushing here, unfortunately.